scripts/core/usm.yml

```diff
@@ -132,14 +132,17 @@ members:
     - type: "size_t"
       name:  maxPoolSize
       desc: "[in] Maximum size of a memory pool"
+    - type: "size_t"
+      name:  maxPoolSizePerDevice
+      desc: "[in] Maximum size of a memory pool per device"
     - type: "size_t"
       name:  maxPoolableSize
       desc: "[in] Allocations up to this limit will be subject to pooling"
     - type: "size_t"
-      name:  capacity
-      desc: "[in] When pooling, each bucket will hold a max of 4 unfreed slabs"
+      name:  bucketCapacity
+      desc: "[in] Maximum number of unfreed allocations in each bucket"
```

Comment:
what do you mean by unfreed allocations?
This should be in bytes or pages.
And even then, I'm not sure how practical this is. Enforcing a single limit for all types of buckets might not make much sense. 4 pages is plenty for 16 byte objects, but very little for 2kb ones.

Reply:
That's true, perhaps this should be a percentage then?

Reply:
I think we just shouldn't have this. I'm not sure how changing bucket sizes is meant to be observable by actual applications or how developers are intended to pick a correct value for this setting without any runtime statistics that can be used to deduce fragmentation per bucket.

Reply:
Right, I'm fine with removing this but I'm wondering if we need to keep backward compatibility. 

Perhaps we could just allow setting this as part of `UR_LEVEL_ZERO_USM_CONFIG` env variable (we could make this env variable have the same format as the env variable in SYCL) and remove it from the programmatic API?

Reply:
If we absolutely need to have it, I would leave it in the API but specify that it's a hint, and implementations are free to ignore it.

Reply:
I removed this: turns out this option was only part of a DEBUG/EXPERIMENTAL API, so we don't need to keep it.